pom.xml

```diff
@@ -28,7 +28,7 @@
     <revision>1.23.0</revision>
     <changelist>-SNAPSHOT</changelist>
     <java.level>8</java.level>
-    <jenkins.version>2.150.3</jenkins.version> <!-- Should be kept in sync with the version resulting from Dockerfile FROM statement-->
+    <jenkins.version>2.176.3</jenkins.version> <!-- Should be kept in sync with the version resulting from Dockerfile FROM statement-->
```

Comment:
Does this need to be updated anywhere else? Maybe here: https://github.com/jenkinsci/blueocean-plugin/blob/12183b7ae4befff6a4eb9158f69835b6d9712d14/Jenkinsfile#L31?

Reply:
This was a mistake.